pkg/utils/sys_schema.go

```diff
@@ -29,11 +29,9 @@ type SysSchema struct {
 	cache *ttlcache.Cache
 }
 
-func NewSysSchema() *SysSchema {
-	c := ttlcache.NewCache()
-	c.SkipTTLExtensionOnHit(true)
+func NewSysSchema(cache *ttlcache.Cache) *SysSchema {
 	return &SysSchema{
-		cache: c,
+		cache: cache,
```

Comment:
This may be wrong since multiple NewSysSchema will share the same cache and cause cache entity conflicts?

Reply:
Introduce lifecycle hooks is a good idea.